docs/mkdocs.yml

```diff
@@ -56,7 +56,11 @@ plugins:
   - autorefs
   - redirects:
       redirect_maps:
+        'cloud/index.md': 'concepts/index.md#langgraph-platform'
+        'cloud/how-tos/index.md': 'how-tos/index.md#langgraph-platform'
         'cloud/concepts/api.md': 'concepts/index.md#langgraph-platform'
+        'cloud/concepts/cloud.md': 'concepts/langgraph_cloud.md'
```

Comment:
does this not use the architecture picture?

Reply:
it does, good catch

Reply:
actually nevermind, i was correct, there was a separate image for this, this one was unused